modul/highscore.py

```
@@ -76,51 +77,57 @@ def update(self, events):
         return None
 
     def draw(self, screen):
-        overlay = pygame.Surface((SCREEN_WIDTH, SCREEN_HEIGHT), pygame.SRCALPHA)
+        overlay = pygame.Surface((C.SCREEN_WIDTH, C.SCREEN_HEIGHT), pygame.SRCALPHA)
         overlay.fill((0, 0, 0, 200))
         screen.blit(overlay, (0, 0))
 
-        title_text = self.font.render("NEW HIGHSCORE!", True, pygame.Color("white"))
+        try:
+            from modul.i18n import gettext
+        except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)